### PR TITLE
Add OpenCompass doc

### PR DESCRIPTION
**Pull request by noemotiovon**
Add OpenCompass doc

### CODE REVIEW

**Comment by noemotiovon** [comment]
There might be some issues that @statelesshz needs to handle. 
When running these scripts on the Ascend server, the device is CPU instead of NPU.

File: sources/opencompass/install.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  CANN 最低版本为 8.0.rc1，安装 CANN 时，请同时安装 Kernel 算子包。
+
+Python 环境创建
+----------------------
+
+.. code-block:: shell
+    :linenos:
+  
+    # 创建 python 3.10 的虚拟环境
+    conda create -y -n openclip python=3.10
```
Comment (MengqingCao) [inline sources/opencompass/install.rst:21]: opencompass

File: sources/opencompass/install.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+----------------------
+
+使用以下 Python 脚本对 open_clip 的安装进行校验，正确打印 open_clip 的版本号和 NPU 卡号说明安装成功。
+
+.. code-block:: python
+    :linenos:
+    :emphasize-lines: 2
+
+    import torch
+    import torch_npu
```
Comment (MengqingCao) [inline sources/opencompass/install.rst:71]: 原生支持npu，这里应该不用导入吧？

File: sources/opencompass/quick_start.rst [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+在 OpenCompass 中评估一个模型通常包括以下几个阶段：配置 -> 推理 -> 评估 -> 可视化。
+
+配置：这是整个工作流的起点。您需要配置整个评估过程，选择要评估的模型和数据集。此外，还可以选择评估策略、计算后端等，并定义显示结果的方式。
+
+推理与评估：在这个阶段，OpenCompass 将会开始对模型和数据集进行并行推理和评估。推理阶段主要是让模型从数据集产生输出，而评估阶段则是衡量这些输出与标准答案的匹配程度。这两个过程会被拆分为多个同时运行的“任务”以提高效率，但请注意，如果计算资源有限，这种策略可能会使评测变得更慢。如果需要了解该问题及解决方案，可以参考
+`FAQ: 效率： <https://opencompass.readthedocs.io/en/latest/get_started/faq.html#efficiency>`_
+
+可视化：评估完成后，OpenCompass 将结果整理成易读的表格，并将其保存为 CSV 和 TXT 文件。你也可以激活飞书状态上报功能，此后可以在飞书客户端中及时获得评测状态报告。
+
+接下来，我们将展示 OpenCompass 的基础用法，展示基座模型模型 ``InternLM2-1.8B`` 和对话模型 ``InternLM2-Chat-1.8B``、``Qwen2-1.5B-Instruct`` 在 GSM8K 和 MATH 下采样数据集上的评估。它们的配置文件可以在 ``configs/eval_chat_demo.py`` 和 configs/eval_base_demo.py 中找到。
```
Comment (MengqingCao) [inline sources/opencompass/quick_start.rst:21]: configs/eval_base_demo.py 加上 ``

Comment (MengqingCao) [reply sources/opencompass/quick_start.rst:21]: 基座模型

**Review by FFFrog — APPROVED** [review]

**Pull request merged**